smartsim/_core/launcher/dragon/dragonLauncher.py

```diff
@@ -144,6 +145,10 @@ def start(
         res = _assert_schema_type(self._connector.send_request(req), DragonRunResponse)
         return LaunchedJobID(res.step_id)
 
+    def get_status(self, *launched_ids: LaunchedJobID) -> tuple[SmartSimStatus, ...]:
+        infos = self._get_managed_step_update(list(launched_ids))
```

Comment:
infos plural intentional?

Reply:
`infos`, plural intentional! The return type of `_get_managed_step_update` is `list[StepInfo]`, so I pluralized as a reminder that this is a "collection of many `StepInfo` instances".

If its unclear or confusing to read, I'm more than willing to rename to `info_list` or similar!